index.js

```diff
@@ -35,7 +35,9 @@ for (var i = 0; i < filesLoadedBeforeTrace.length; i++) {
   var matches = moduleRegex.exec(filesLoadedBeforeTrace[i]);
   if (matches && matches.length > 1 &&
       modulesLoadedBeforeTrace.indexOf(matches[1]) === -1) {
-    modulesLoadedBeforeTrace.push(matches[1]);
+    if (matches[1] !== '@google' && matches[1] !=='cloud-trace') {
```

Comment:
@matthewloring I added 'cloud-trace' as i was unsure how older node versions would handle the npm package orgs, I'll remove and we'll see on Travis I guess.